templates/frontend/edid_upload_text.html

```diff
@@ -52,6 +52,11 @@ <h4>Thanks!</h4>
                 <input type="radio" id="id_text_type_xrandr" name="text_type" value="xrandr"{% ifequal form.text_type.value 'xrandr' %} checked{% endifequal %}> XRandR
               </label>
               <p>Linux users, run <code>xrandr --props</code> and paste its output.</p>
+              <label class="radio">
+                <input type="radio" id="id_text_type_xoglog" name="text_type" value="xorglog"{% ifequal form.text_type.value 'xorglog' %} checked{% endifequal %}> Xorg.log
```

Comment:
"id_text_type_xoglog" maybe meant ...xorglog?

Reply:
Maybe :) Fixed, but FWIW I don't think this has any practical implications on implementation.